.github/workflows/kind.yml

```
@@ -102,6 +102,8 @@ jobs:
         deployOnHub: ${{ matrix.deployOnHub }}
         HOSTED: ${{ matrix.hosted }}
       run: |
+        export CALLER_REPO=${{ github.event.organization.login }}
+
```

Nit: If the `github` context is available there, I think this would be cleaner in the `env` rather than as a separate export.
```suggestion
        deployOnHub: ${{ matrix.deployOnHub }}
        HOSTED: ${{ matrix.hosted }}
        CALLER_REPO: ${{ github.event.organization.login }}
      run: |
```